taipy/gui/gui.py

```diff
@@ -965,15 +965,18 @@ def _serve_status(self, template: Path) -> t.Dict[str, t.Dict[str, str]]:
 
     def __upload_files(self):
         self.__set_client_id_in_context()
-        if "var_name" not in request.form:
+        on_upload_action = request.form.get("on_action", None)
+        var_name = request.form.get("var_name", None)
+        if not var_name and not on_upload_action:
             _warn("No var name")
```

Comment:
```suggestion
            _warn("__upload_files(): No var name")
```
or any other way to locate the warning source